client/src/legacy/BootstrapComponent.js

```diff
@@ -0,0 +1,48 @@
+/* global ss */
+import jQuery from 'jquery';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { loadComponent } from 'lib/Injector';
+
+jQuery.entwine('ss', ($) => {
```

Comment:
This bit uses entwine to wire our react component. All Component based off the BootstrapComponent trait would use this generic bootstrapping logic.

Reply:
looks good - quite similar to our implementation here: https://github.com/silverstripeltd/tourism1/blob/master/app/javascript/src/entwine/reactFieldBootstrapper.js we've got some extra handling for having an input field that the react component updates